internal/remote/skr_context_provider.go

```diff
@@ -63,7 +69,7 @@ func (k *KymaSkrContextProvider) Init(ctx context.Context, kyma types.Namespaced
 		return fmt.Errorf("failed to create lookup client: %w", err)
 	}
 
-	skrClient := NewClientWithConfig(clnt, restConfig)
+	skrClient := clnt
```

Comment:
`skrClient` should be the var of the already above created client, and this statement removed.

Reply:
[a9e9233](https://github.com/kyma-project/lifecycle-manager/pull/2877/commits/a9e9233df6d1da423272b44d3d15aaafacd5c9df)